cmd/mimir/config-descriptor.json

```
@@ -3063,6 +3074,17 @@
           "fieldType": "duration",
           "fieldCategory": "experimental"
         },
+        {
+          "kind": "field",
+          "name": "separate_metrics_label",
+          "required": false,
+          "desc": "Label used to further separate specific distributor and ingester metrics adding a 'group' label. For each write request, the group is obtained from the first non-empty group label from the incoming list of timeseries.",
```

```suggestion
          "desc": "Separate specific distributor and ingester metrics by adding a `group` label. For each write request, the group comes from the first non-empty group label of the incoming list of timeseries.",
```
Please vet this for technical accuracy. I think there is also more room for improvement here, but I am not close enough to the tech yet to know exactly what else to simplify.